### PR TITLE
Shovels

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Adds more shovel variants to mining and advanced mining techwebs

## Why It's Good For The Game

ok digging is actually fine

## Changelog
:cl:
add: Added advanced shovels to mining related techwebs. More specifically: Titanium shovel, wide titanium shovel, diamond shovel and wide diamond shovel.
/:cl:

### CODE REVIEW

File: modular_skyrat/code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -56,14 +71,10 @@
 	prereq_ids = list("advanced_illegal_ballistics", "adv_mecha", "adv_mecha_tools", "mech_carbine")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 12500)
 
-/datum/techweb_node/syndicate_basic/New()
```
Comment (jjpark-kb) [inline modular_skyrat/code/modules/research/techweb/all_nodes.dm:59]: why is this being removed?

File: modular_skyrat/code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -56,14 +71,10 @@
...
-	design_ids += "armblade"
-	. = ..()
-
```
Comment (jjpark-kb) [inline modular_skyrat/code/modules/research/techweb/all_nodes.dm:62]: Why is this being removed?

Comment (BobJoga) [reply modular_skyrat/code/modules/research/techweb/all_nodes.dm:62]: it wasn't supposed to be, wtf

Comment (BobJoga) [reply modular_skyrat/code/modules/research/techweb/all_nodes.dm:62]: wait it's fine i just moved it up a bit

**Pull request merged**